lib/translate/lun_translator.h

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef LIB_LUN_TRANSLATOR_H
+#define LIB_LUN_TRANSLATOR_H
+
+#include <optional>
+
+#include "../../third_party/spdk_defs/nvme_defs.h"
+#include "../scsi_defs.h"
+#include "absl/types/span.h"
+#include "common.h"
+
+namespace translator {
+
+std::optional<nvme_defs::GenericQueueEntryCmd> translateReportLunCommand(
```

Comment:
Functions (here and below) should be named consistent with [the relevant section of the style guide](https://google.github.io/styleguide/cppguide.html#Function_Names).